android/KioskActivity.java

```diff
@@ -65,7 +65,7 @@ public void onCreate(Bundle savedInstanceState) {
         if (actionBar != null) actionBar.hide();
 
         // add overlay to prevent statusbar access by swiping
-        statusBarOverlay = StatusBarOverlay.createOrObtainPermission(this);
+        // statusBarOverlay = StatusBarOverlay.createOrObtainPermission(this);
```

Comment:
It would be better to just delete it. It's still in the commit history. You can easily revert it. But having unused code commented out in your production code isn't a good practice.